### PR TITLE
Upload gzipped files

### PR DESCRIPTION
**Pull request by mstajdohar**

### CODE REVIEW

File: resolwe_api/resolwe.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 """Resolwe"""
-from __future__ import absolute_import, division, print_function, unicode_literals
```
Comment (kostko) [inline resolwe_api/resolwe.py]: Did you find out what exactly causes the crash in case `unicode_literals` are used?

**Comment by kostko** [comment]
LGTM

File: resolwe_api/resolwe.py (hunk 1 above)
Comment (dblenkus) [inline resolwe_api/resolwe.py]: Does this also work in py3?

Comment (mstajdohar) [inline resolwe_api/resolwe.py]: @kostko: no, @dblenkus: yes

**Pull request merged**